From 6c7ad14963b40a6462ea8f1c66b7321c60bca8f2 Mon Sep 17 00:00:00 2001
From: Jonathan M Hsieh <jon@cloudera.com>
Date: Thu, 11 Feb 2016 17:52:28 -0800
Subject: [PATCH 212/217] CDH-37191
 org.apache.hadoop.hbase.spark.TestJavaHBaseContext
 unit tests failing on cdh5-1.2.0

- Remove JavaSparkContext.addJar("spark.jar") - causes unit test failure

Change-Id: I56470093265c19767cecaa02eae61a80e4d45102
Reason: Bug
Author: Joanthan Hsieh
Ref: CDH-37191
---
 .../hadoop/hbase/spark/TestJavaHBaseContext.java   |    1 -
 1 file changed, 1 deletion(-)

diff --git a/hbase-spark/src/test/java/org/apache/hadoop/hbase/spark/TestJavaHBaseContext.java b/hbase-spark/src/test/java/org/apache/hadoop/hbase/spark/TestJavaHBaseContext.java
index 724ac36..6042998 100644
--- a/hbase-spark/src/test/java/org/apache/hadoop/hbase/spark/TestJavaHBaseContext.java
+++ b/hbase-spark/src/test/java/org/apache/hadoop/hbase/spark/TestJavaHBaseContext.java
@@ -65,7 +65,6 @@ public class TestJavaHBaseContext implements Serializable {
   @Before
   public void setUp() {
     jsc = new JavaSparkContext("local", "JavaHBaseContextSuite");
-    jsc.addJar("spark.jar");
 
     File tempDir = Files.createTempDir();
     tempDir.deleteOnExit();
-- 
1.7.9.5

